packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_model.dart

```diff
@@ -35,15 +59,17 @@ class LoggingTableModel extends ChangeNotifier with DisposerMixin {
       preferences.logging.retentionLimit,
       _onRetentionLimitUpdate,
     );
+
+    _retentionLimit = preferences.logging.retentionLimit.value;
+    subscribeToFilterChanges();
   }
 
-  final _logs = ListQueue<LogDataV2>();
-  final _filteredLogs = ListQueue<LogDataV2>();
+  final _logs = ListQueue<_LogEntry>();
+  final _filteredLogs = ListQueue<_FilteredLogEntry>();
```

Comment:
can you add a comment explaining why we are storing a secondary data set for filtered logs in addition to the `filteredData` provided by the FilterControllerMixin